static/data/hierarchy_statsvar.json

```diff
@@ -127217,7 +127217,8 @@
      }
     ]
    }
-  ]
```

Comment:
there is no pv tree code change, are these from the run of the latest pv tree tool?

Reply:
I dropped the pv tree files from this PR